HHS/Head-Start-TTADP
# Add review page accordions

**Description of change**

The review page now has an accordion with an item for each page that matches the desired design. The report still needs to be marked as `Submitted` in the backend after saving of a report (https://github.com/HHS/Head-Start-TTADP/issues/98) is closed.

 * Each page defines sections that are used in the review accordion
 * Simplified how the activity report imports the pages for the report
 * Date picker now accepts a string MM/DD/YYYY and spits out the same
 * Activity report page file names are no longer capitalized because they are no longer react components

**How to test**

Dev:

Test on https://tta-smarthub-dev.app.cloud.gov/activity-reports/review. Note I haven't tested on DEV yet because I can't login to the UAT environment. I didn't want to wait for UAT to work before getting eyes on this PR.

Local:

1. Pull down
2. Run migrations and seeders
3. Go to Activity report and fill out a report
4. Click the accordion in the review page, verify form values are correctly shown

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/135
* closes https://github.com/HHS/Head-Start-TTADP/issues/245

**Checklist**

- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
